setup.cfg

```diff
@@ -58,6 +58,8 @@ install_requires =
     requests>=2.13.0,<3.0.0
     pydantic>=1.7.4,!=1.8,!=1.8.1,<1.10.0
     jinja2
+    rapidfuzz>=2.4.0,<3.0.0
+    rapidfuzz_capi>=1.0.5,<2.0.0
```

Comment:
`rapidfuzz_capi` needs to be placed in `setup_requires` similar to numpy when compiling against the C-API

Reply:
(But note that while we maintain `setup_requires` here for historical/unclear reasons, it doesn't actually work in `setup.cfg` and the place that actually matters is `pyproject.toml`.)

Reply:
Oh I did not see the existence of the pyproject.toml file. Yes that is definetly the correct place. Then it is simply not required in setup.cfg

Reply:
tests fail if setup.cfg is not identical to requirements.txt: https://github.com/explosion/spaCy/blob/master/spacy/tests/package/test_requirements.py#L66-L68

build instructions specify `pip install -r requirements.txt`, not `poetry install`: https://github.com/explosion/spaCy/blob/master/spacy/tests/package/test_requirements.py#L66-L68

Reply:
Ok. This should be added to the `pyproject.toml` anyways